gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -114,11 +131,18 @@ final void createManifest() throws IOException {
 
         List<ProductDependency> productDependencies = productDependencyManifest.productDependencies();
         if (productDependencies.isEmpty()) {
-            requireAbsentLockfile();
+            requireAbsentLockfile(getLockfile());
         } else {
             ensureLockfileIsUpToDate(productDependencies);
```

Comment:
small: Can we rename this method to something like `ensurePdepsLockfileIsUpToDate` to make it clear that we now manage two different lock files?